extensions/2.0/Khronos/KHR_spz_gaussian_splats_compression/README.md

```diff
@@ -0,0 +1,416 @@
+| Color (Spherical Harmonic degree 0 (Diffuse) and alpha) | COLOR_0 | VEC4 | unsigned byte normalized or float | yes | |
+| Rotation | _ROTATION | VEC4 | float | yes | Rotation is a quaternion. |
+| Scale | _SCALE | VEC3 | float | yes | |
+| Spherical Harmonics degree 1 | _SH_DEGREE_1_COEF_n (n = 0 to 2) | VEC3 | float | no (yes if degree 2 or 3 are used) | |
+| Spherical Harmonics degree 2 | _SH_DEGREE_2_COEF_n (n = 0 to 4) | VEC3 | float | no (yes if degree 3 is used) | |
+| Spherical Harmonics degree 3 | _SH_DEGREE_3_COEF_n (n = 0 to 6) | VEC3 | float | no | |
+
+Each increasing degree of spherical harmonics requires more coeffecients. At the 1st degree, 3 sets of coeffcients are required, increasing to 5 sets for the 2nd degree, and increasing to 7 sets at the 3rd degree. With all 3 degrees, this results in 45 spherical harmonic coefficients stored in the `_SH_DEGREE_ℓ_COEF_n` attributes.
+
+**TODO:** SPZ to attributes diagram
+
+### Accessors
+
+Required `accessors` for `POSITION`, `COLOR_0`, `_ROTATION`, and `_SCALE`:
+
+```json
+  "accessors": [{
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC3",
+      "max": [
+        1,
+        1,
+        1,
+      ],
+      "min": [
+        -1,
+        -1,
+        -1,
+      ]
+    }, {
+      "componentType": 5121,
+      "count": 590392,
+      "type": "VEC4",
+      "normalized": true
+    }, {
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC3"
+    }, {
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC4"
+    }],
+```
+
+Spherical harmonics `accessors` all follow the pattern:
+
+```json
+  "accessors": [{
+    "componentType": 5126,
+    "count": 590392,
+    "type": "SCALAR"
+  }]
+```
+
+At minimum accessors will be defined for `POSITION`, `COLOR_0`, `_ROTATION`, and `_SCALE`. Each must have `componentType`, `count`, and `type` defined.
+
+Accessor `type` is defined for the resulting type after decompression and dequantization has occurred.
+
+The accessor `count` must match `numPoints` in `KHR_spz_gaussian_splats_compression`.
+
+## Conformance
+
+The recommended process for handling SPZ compression is as follows:
+
+- If the loader does not support `KHR_spz_gaussian_splats_compression`, it must fail.
+- If the loader does support `KHR_spz_gaussian_splats_compression` then:
+
+  - The loader must process `KHR_spz_gaussian_splats_compression` data first. The loader must get the data from `KHR_spz_gaussian_splats_compression`'s `bufferView` property.
+  - The loader then must process `attributes` of the `primitive`. When processing the loader must ignore any `bufferView` and `byteOffset` in the `accessor` and instead use values derived from the decompressed data streams. This data can be used to populate the `accessors` or render directly.
+  - Any attributes not listed in `KHR_spz_gaussian_splats_compression`'s `attributes` but are listed in the `primitive` must then be processed by the loader.

```

Comment:
The extension object no longer has `attributes` so maybe this note can be removed?

Reply:
Yes. Instead, we probably need to mention here how the mapping should occur. Adding this now.